tests/unit_tests.cpp

```diff
@@ -151,9 +151,24 @@ TEST_CASE("Quotation marks")
   CHECK_EQ(res[4], "44.29124");
   CHECK_EQ(res[5], "1");
 }
+
+TEST_CASE("Not wrapped quotation marks")
+{
+  const auto res = CsvParser::split_record(R"(Contains "quotes", commas and text)");
+  REQUIRE_EQ(res.size(), 2);
+  CHECK_EQ(res[0], R"(Contains "quotes")");
+  CHECK_EQ(res[1], "commas and text");
+}
+
+TEST_CASE("Wrapped quotation marks")
+{
+  const auto res = CsvParser::split_record(R"("Contains ""quotes"", commas and text")");
+  REQUIRE_EQ(res.size(), 1);
+  CHECK_EQ(res[0], R"(Contains "quotes", commas and text)");
+}
 TEST_SUITE_END();
```

Comment:
```suggestion

TEST_CASE("Read quoted empty values")
{
  const auto res = CsvParser::split_record(",\"\"");
  REQUIRE_EQ(res.size(), 2);
  CHECK_EQ(res[0], "");
  CHECK_EQ(res[1], "");
}

TEST_SUITE_END();
```
Функция unquote_text пустые значения в кавычках не умеет распаковывать.